Client/Frontend/Browser/TabsBar/TabBarCell.swift

```diff
@@ -126,4 +129,12 @@ class TabBarCell: UICollectionViewCell {
             strongSelf.titleLabel.text = tab.displayTitle
         }
     }
+    
+    func applyTheme(_ theme: Theme) {
+        styleChildren(theme: theme)
+        
```

Comment:
Tiny nit: calling `styleChildren` here is not required since you have not assigned `themableChildren` to anything